docs/changes/0.19.0.rst

```diff
@@ -0,0 +1,79 @@
+Changelog for QCoDeS 0.19.0
+===========================
+
+The October 2020 release of QCoDeS.
+
+
+Breaking Changes:
+_________________
+
+There are no breaking changes in this release of QCoDeS.
+
+
+New:
+____
+- Keysight B1511B SMU driver has been added for support with the B1500 (#2278)
+- Notebook: Example of how buffered sweep could be done on param with setpoints (#2263)
+
+
+Improved:
+_________
+- Station: Take verbose error message when validating station yaml against schema (#2282)
+- Add optional shape metadata to dataset (#2145)
+
+
+Improved drivers:
+_________________
+- AMI430: `has_current_rating` has been removed in the driver to work with the new firmware version (#2291)
+- Small typo fix in alazer drivers (#2296) 
+- Yokogawa GS200: `source_mode` bug fix (#2143)
+- Keysight B1517A: Improvments for valid voltage and current output ranges (#2278)
+- Alazer 9400:(#2301)
+  - Remove redundant get_cmd from all parameters
+  - Remove redundant set_cmd from `TraceParameters`
+  - Rewrite loops over channels and trigger engines to be more explicit
+  - Make dll_path an argument
+
+
+Under the hood:
```

Comment:
I've left out the PR for updates needed with pyvisa 1.11 since it hasn't laded yet. I'll add that once it does.

Reply:
already added, in advance 👍